compiler/test/stdlib/number.test.gr

```diff
@@ -639,9 +651,9 @@ assert Number.parse("-3/-9") == Ok(1/3)
 assert Number.parse("0x3/-9") == Ok(-1/3)
 assert Number.parse("3/-0x9") == Ok(-1/3)
 assert Number.parse("9223372036854775808/27_670_116_110_564_327_424") == Ok(1/3)
-assert Number.parse("1/2/") == Err("Invalid digit in input")
-assert Number.parse("1/") == Err("Invalid input")
-assert Number.parse("1//") == Err("Invalid digit in input")
+assert Result.isErr(Number.parse("1/2/"))
+assert Result.isErr(Number.parse("1/"))
+assert Result.isErr(Number.parse("1//"))
```

Comment:
Maybe we should check for the specific error here?

Reply:
I test some of the cases after doing the generic error tests.